services/skus/storage/repository/repository.go

```diff
@@ -142,6 +142,22 @@ func (r *Order) GetTimeBounds(ctx context.Context, dbi sqlx.QueryerContext, id u
 	return result, nil
 }
 
+// GetExpiresAtP1M returns expires_at that is last_paid_at (or now()) plus 1 month.
+func (r *Order) GetExpiresAtP1M(ctx context.Context, dbi sqlx.QueryerContext, id uuid.UUID) (time.Time, error) {
+	const q = `SELECT (SELECT COALESCE(last_paid_at, now()) AS last_paid_at) + interval '1 month' AS expires_at FROM orders WHERE id = $1`
```

Comment:
Could we use the (max) value in the order item list for valid_for_iso instead of having a per interval function to make it more generic?

`max(order_items.valid_for_iso)::interval` ??